src/Raven.Studio/typescript/commands/resources/saveDatabaseSettingsCommand.ts

```diff
@@ -0,0 +1,23 @@
+import commandBase = require("commands/commandBase");
+import database = require("models/resources/database");
+import endpoints = require("endpoints");
+
+class saveDatabaseSettingsCommand extends commandBase {
```

Comment:
maybe `createDatabaseCommand` ? It is called like that in the Client API

Reply:
(talked with @ppekrol about that personally) - solved